tests/core/test_structured_data_signing.py

```diff
@@ -809,22 +812,18 @@ def test_type_regex_for_redos():
     assert done < 1
 
 
-def test_structured_data_invalid_identifier_filtered_by_regex():
```

Comment:
I'm not sure what these tests were for, are they no longer valid?

Reply:
They were 2 tests, one each for a space in a name and a space in a type. I can add a few more what what are now the "invalid" characters like symbols. Unescaped quotes errors even as the json is being loaded, will look at test there too.